Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,21 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes/archive/v2.2.2.tar.gz"
+  sha256 "31d5bec7084793f3114119826d961bb85cd0647719587c4b0429a0e7bd4f4495"
+
+  depends_on "tmux" => :recommended
+
+  def install
+    system "./install.sh", prefix
+  end
+
+  test do
+    # Check options with valid combination
+    system "echo testarg | #{bin}/xpanes --dry-run -c echo"
```

Comment:
Use `pipe_output()`.

Reply:
wow, it's helpful 😄  thx.
Fixed (387b2d8cef2aefda80226f0aa9e7d8037db293e4)